### ISSUE TITLE
Provide Empty state for when there are no results

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#664]
## Description

When there are no results in the filter, there is no empty state to allow the user to know that the page is blank because there is nothing there.  We should show an empty state, and announce this state to screen reader users.

<img width="1058" alt="Screenshot 2024-05-30 at 12 13 38" src="https://github.com/CodeForPhilly/clean-and-green-philly/assets/102739972/05472883-9318-4805-862b-c400bfe7c0a2">

## Acceptance Criteria

- [ ] Show an empty state (ex. No results found) when there are no results after applying filters.
- [ ] Use `aria-live` region to announce the state

**Comment by KenMain5** [comment]
Can I get assigned to this task?

**Comment by bacitracin** [comment]
@KenMain5 assigned!

**Comment by KenMain5** [comment]
Is there anyone I can talk to on how to design this? or what it should look etc?

**Comment by CodeWritingCow** [comment]
> Is there anyone I can talk to on how to design this? or what it should look etc?

HI @thansidwell and @marvieqa, do you have wireframes or suggestions for how this should look like in the UI?

**Comment by thansidwell** [comment]
@CodeWritingCow @marvieqa Yes! Here's the design. https://www.figma.com/design/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?node-id=6265-35268&t=B1Q1D5C9HzoiZWLE-1

**Comment by KenMain5** [comment]
mb I've been busy, I'll get it done this weekend!

**Comment by CodeWritingCow** [comment]
@KenMain5 no worries! You have 2 other tickets on your plate already. If you feel like you're taking on more than your time capacity allows, I can take this off your plate so you can focus on the other tickets. Also, @AZBL is looking to pick up a ticket, so maybe you and Alex can chat on Slack and work something. More importantly, you should be able to enjoy your weekend!

**Comment by KenMain5** [comment]
@CodeWritingCow okay I'll work something out with him!

**Comment by CodeWritingCow** [comment]
Per conversation with @KenMain5, I've unassigned Kenny from this ticket. He's working on other tickets, so this one is open for other developers to take.

**Comment by Michael-Dratch** [comment]
Hello, can I pick up this issue?

**Comment by CodeWritingCow** [comment]
@Michael-Dratch it's yours!
